examples/phi2/phi2.py

```diff
@@ -35,7 +36,14 @@ def get_args(raw_args):
 
 
 def main(raw_args=None):
-    if version.parse(OrtVersion) < version.parse("1.17.0"):
+    if platform.system() == "Linux" and version.parse(OrtVersion) < version.parse("1.18.0"):
+        raise ValueError(
+            "Please use onnxruntime>=1.18.0 for phi2 optimization in Linux, you can refer to "
```

Comment:
it need nightly build?

Reply:
yes, https://github.com/microsoft/Olive/tree/main/examples/phi2#prerequisites

Reply:
Yes, I checked the `phi` model type and found it is only in nightly package. 
For example, in latest 1.17.2, there is not `phi`
https://github.com/microsoft/onnxruntime/blob/rel-1.17.2/onnxruntime/python/tools/transformers/optimizer.py#L45C1-L61C2